src/battle/overlay_12_0224E4FC.c

```diff
@@ -1770,3 +1770,195 @@ void ov12_02251038(BattleSystem *bsys, BATTLECONTEXT *ctx) {
     ctx->unk_311C = 6;
     ctx->unk_311D = 6;   
 }
+
+void InitSwitchWork(BattleSystem *bsys, BATTLECONTEXT *ctx, int battlerId) {
+    int i;
+    int maxBattlers;
+    u8 *data;
+    UnkBattlemonSub unkStruct = ctx->battleMons[battlerId].unk88;
+    
+    maxBattlers = BattleSys_GetMaxBattlers(bsys);
+    BattleSys_GetBattleType(bsys);
+    ctx->unk_21A8[battlerId][0] = 40;
+    
+    if (!(ctx->linkStatus & (1 << 8))) { //not baton pass
```

Comment:
constants?

Reply:
still unsure if linkStatus is the right term, original is referred to as 'server_status' but it could mean in the server-client type instead of the link type, so holding off naming consts until I get more info on it